docs/manual/hybrid_apps.rst

```diff
@@ -0,0 +1,47 @@
+Hybrid Apps
+===========
+
+.. note::
+
+    Check out the `Hybrid App Sample Project <https://github.com/GodotVR/godot_openxr_vendors/tree/master/samples/hybrid-app-sample>`_
+    for a working demo of hybird app functionality.
+
+Hybrid apps are applications capable of swapping between an immersive 3D mode and a 2D panel mode at runtime.
+
+Project Settings
+----------------
+
+To enable hybrid app functionality, the feature must be enabled in your project settings. The setting can be found in **Project Settings** under the **OpenXR** section.
```

Comment:
Just to avoid saying "enable" so many times:

```suggestion
To make a hybrid app, the feature must be enabled in your project settings. The setting can be found in **Project Settings** under the **OpenXR** section.
```